app/Actions/Diagnostics/Checks/IniSettingsCheck.php

```diff
@@ -68,7 +68,7 @@ public function check(array &$errors): void
 			$errors[] = 'Warning: user_agent for PHP is not set. You may experience problems when importing images via URL.';
 		}
 
-		if (!(ini_get('assert.exception') === '1')) {
+		if (ini_get('assert.exception') !== '1') {
 			$errors[] = 'Warning: assert.exception is set to false. Lychee assumes that failing assertions throw proper exceptions.';
 		}
```

Comment:
I wonder whether this is still true. Do we really rely on that? Of course this is not part of this PR and maybe should become its own issue to check whether some of the diagnostic is still meaningful.